syntaxes/omt.tmLanguage.json

```diff
@@ -711,7 +726,14 @@
             "patterns": [
                 { "include": "#comments" },
                 {
-                    "begin": "^( +)(\\w+):(.*)$",
+                    "match": "^ +(\\w+): +\\{}(.*)$",
+                    "captures": {
+                        "1": { "name": "support.function.omt" },
+                        "2": { "patterns": [{ "include": "#_comments-only" }] }
+                    }
+                },
+                {
+                    "begin": "^( +)(\\w+):(?! +\\{})(.*)$",
```

Comment:
why capture the first group if you are not going to use it?

- [ ] remove the first capture group

Reply:
Same situation here: the first capture group is used in the end regex.